cypress/e2e/14-timeline/timeline.spec.ts

```diff
@@ -151,6 +151,21 @@ describe("Timeline", () => {
             cy.percySnapshot("Configured room on IRC layout");
         });
 
+        it("should add inline start padding to EventTile_line of GenericEventListSummary on IRC layout", () => {
+            cy.visit("/#/room/" + roomId);
+            cy.setSettingValue("layout", null, SettingLevel.DEVICE, Layout.IRC);
+
+            // cf. _IRCLayout.pcss
+            //  --name-width: 80px;
+            //  --icon-width: 14px;
+            //  --right-padding: 5px;
+            // cf. _EventTile.pcss
+            //  calc(var(--name-width) + var(--icon-width) + $MessageTimestamp_width + 3 * var(--right-padding));
+            //  = 80 + 14 + 46($MessageTimestamp_width) + 3*5 = 155px
+            cy.get(".mx_GenericEventListSummary[data-layout=irc] > .mx_EventTile_line")
+                .should('have.css', 'padding-inline-start', '155px');
```

Comment:
Setting this value to `145px` (`--name-width`: 70px) returns an error, therefore `--name-width` on _IRClayout.pcss should be changed from 70px to 80px.

Reply:
If only CSS properties are asserted without a screenshot this test should be a jest unit-test. You can ask for support on how to do this through the usual channels.